bug/op_add_comment.go

```diff
@@ -30,22 +30,19 @@ func (op *AddCommentOperation) Id() entity.Id {
 func (op *AddCommentOperation) Apply(snapshot *Snapshot) {
 	snapshot.addActor(op.Author_)
 	snapshot.addParticipant(op.Author_)
```

Comment:
Consideration:
- Why is this done within this operation? Can't Snapshot::Post(Comment) not set the Author internally to Participants, Actor etc.?
  - Solution: Move over to dedicated Snapshot methode. But then other operations would then require long parameter lists which seems to be unreasonable. 
    E.g. ChangeTitle(title string, Author Identity.Interface, changedAt timestamp.Timestamp)
  - Solution: Maybe participant and actors shouldn't be part of a bug. Instead they should be stored within an Identity.